src/v2/view-builder/views/EnrollProfileView.js

```diff
@@ -9,6 +9,15 @@ const Body = BaseForm.extend({
 
   save () {
     return loc('registration.form.submit', 'login');
+  },
+
+  saveForm () {
+    this.settings.preSubmit(this.model.attributes,
```

Comment:
`this.model.toJSON()` ?

Reply:
Copying v1 here

Reply:
I am not sure which value would be more helpful for a user of this hook. I think they'd want to handle the raw model attributes.

Reply:
this.model.attributes
```
{
    "formName": "enroll-profile",
    "userProfile.lastName": "asd",
    "userProfile.firstName": "asda",
    "userProfile.email": "sd@gmail.com"
}
```
this.model.toJSON()
```
{
    "userProfile": {
        "lastName": "asd",
        "firstName": "asda",
        "email": "sd@gmail.com"
    }
}
```

Reply:
this.model.toJSON() does look better and they don't need to know the formName

Reply:
> handle the raw model attributes.

hmm, I doubt so.
maybe @nikhilvenkatraman-okta could add more color

Reply:
Good question. We didn't really have form-name in V1 https://github.com/okta/okta-signin-widget/blob/master/test/unit/spec/Registration_spec.js#L745-L804 and it does seem like something internal that we can hide for the end-user. `this.model.toJSON()` might be a better option where they can maybe do something like this ?

OIE 
```
       userProfile.zip = {
              type: 'string',
              title: 'Zip',
              description: 'Zip code',
              default: 'Enter your zip code',
              maxLength: 255
            };
```
V1

```
        schema.profileSchema.properties.zip = {
              type: 'string',
              title: 'Zip',
              description: 'Zip code',
              default: 'Enter your zip code',
              maxLength: 255,
            };

```